app/services/distribution_update_service.rb

```diff
@@ -16,12 +17,21 @@ def call
       distribution.update! @params
       distribution.reload
       @new_issued_at = distribution.issued_at
+      @new_delivery_method = distribution.delivery_method
       # Apply the new changes to the storage location inventory
       distribution.storage_location.decrease_inventory(distribution.to_a)
     end
   end
 
   def resend_notification?
+    issued_at_changed? || delivery_method_changed?
+  end
+
+  def issued_at_changed?
     @old_issued_at.to_date != @new_issued_at.to_date
   end
+
+  def delivery_method_changed?
```

Comment:
nit: these methods aren't used to determine the subject with your latest changes, so maybe they should be private?